projects/extension/src/containers/Options.tsx

```diff
@@ -37,6 +37,13 @@ interface StyledTabProps {
   label: string
 }
 
+type logStructure = {
+  time: string
+  level: string
```

Comment:
If you don't use this field, then don't include it here

Reply:
I am supposed to use it (all logs) - erroneously (copy/paste) left out. Thank you